staging/src/k8s.io/apimachinery/pkg/util/httpstream/spdy/roundtripper.go

```diff
@@ -305,24 +296,6 @@ func (s *SpdyRoundTripper) dialWithoutProxy(ctx context.Context, url *url.URL) (
 	if err != nil {
 		return nil, err
 	}
-
```

Comment:
this is the output of tls.Dial and tls.DialWithDialer , that AFAIK already verify the Hostname during the handshake